### ISSUE TITLE
Cloud Native Ruby on Rails

### ISSUE THREAD

**Issue by wxw-matt** [rails-oceania/roro#304]
Thanks for proposing a talk for RORO Sydney!

Please provide the following information:

- Name of talk
  **Matt Wang**
- Brief summary of the talk content
  > Make the Ruby on Rails application as a Cloud Native application from the start of the development.
     **Present a tool to make this process simple and enjoyable.**
- Which month's meetup you would like to present at
  > I guess the incoming one in Fed.
- Length of talk (please try to keep to ~15 minutes, but we're happy to allow wiggle room; however, the more succinct, the better!)
  > Should be less than 15 mins.
- Your Twitter username (if you have one)
  > I have it and do not really use it. Sorry.
- Are you happy to have your talk published on the RubyAU YouTube channel?
  > Very happy
- Link to, or upload of, a photo of yourself that will appear in the meetup slide deck (if you don't specify one, we'll see if we can find one from your social media accounts)
  > Not sure if you are happy with the quality of this [one](https://miro.medium.com/fit/c/176/176/1*yfbTyMWE_v-swjFFWRoInQ.png)

**Comment by mathewdbutton** [comment]
Hey @wxw-matt this sounds great, we'd love to have you speak! The February meetup is on the 8th, is that still okay for you?

No trouble with the photo, it should work okay.

**Comment by wxw-matt** [comment]
@mathewdbutton Thanks for your quick reply. I guess so. However, just wondering when is the next meetup after the one in Fed.

**Comment by mathewdbutton** [comment]
The next one after Feb will be on March 8th (it's the second Tuesday of every month)

On January 28, 2022 5:27:10 PM GMT+11:00, wxw-matt ***@***.***> wrote:
***@***.*** Thanks for your quick reply. I guess so. However, just wondering when is the next meetup after the one in Fed.

**Comment by mathewdbutton** [comment]
Thank you again for doing this @wxw-matt I'll have the video of the recording up soon. Sorry for the lateness of this, I'll post the link here when it's up.

**Comment by wxw-matt** [comment]
Thanks @mathewdbutton. I have been updating the project, now it can be used to do all development tasks as the `rails` and `rake` command can do. Besides, the most exciting feature is that `rod` can deploy projects to the Kubernetes.
I changed its name to `xod`, because I am going to add support for other languages, such as Python, Golang and Java.
Here is a [video](https://youtu.be/layqHbcxnFw) I made for UNIHACK 2022.
